shared/actions/profile.js

```diff
@@ -404,13 +412,13 @@ function outputInstructionsActionLink (): AsyncAction {
         openURL(`https://coinbase.com/${profile.username}#settings`)
         break
       case 'twitter':
-        openURL(`https://twitter.com/home?status=${profile.proof}`)
+        openURLIfNotNull(profile.proofText, `https://twitter.com/home?status=${profile.proofText || ''}`, 'twitter url')
```

Comment:
I don't think it's that simple.

the caller here doesn't know about profile or proofText, just that it wants to outputInstructionsActionLink. Definitely interesting though, will give it some thought